### PR TITLE
Loader Atmos Upgrade

### PR DESCRIPTION
**Pull request by SurfinNinja1**
God I hope I did this right, I barely know how to PR properly

## About the PR

The Loader class vessels have been outfitted with a more comprehensive atmospherics setup.  There is now atmos in every room aboard the ship.

## Why / Balance

I heard some people were dying to CO2 asphyxiation in the engine room. I fixed it. I immediately followed up by just setting it up in every room after just so it's all done right.
## Technical details

No code changes, just adding spaghetti to the ship.
## Media

- [✅] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

Added: Air vents and scrubbers have been installed in all rooms in the ship.

### CODE REVIEW

**Comment by dvir001** [comment]
Pictures?

File: Resources/Prototypes/_NF/Shipyard/loader.yml [hunk 1]
```diff
@@ -5,19 +5,16 @@
...
   id: Loader
   mapName: 'NC Loader'
-  mapPath: /Maps/Shuttles/loader.yml
   minPlayers: 0
   stations:
     Loader:
       stationProto: StandardFrontierVessel
       components:
         - type: StationNameSetup
-          mapNameTemplate: 'Loader {1}'
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Shipyard/loader.yml:20]: Removing this lines break the ship from ever working, please fix it

**Review by dvir001 — CHANGES_REQUESTED** [review]
Also note the wrong file uploaded.

**Comment by SurfinNinja1** [comment]
im gonna re-do this.... i messed up

**Pull request closed without merging**